lib/diagnostic.rb

```diff
@@ -40,30 +53,34 @@
 
 class Animal
   def initialize
-      @sound = nil
+    @sound = nil
   end
 
   def say_hello
-      puts "I am a #{self.class.name}, and I go '#{@sound}'"
+    puts "I am a #{self.class.name}, and I go '#{@sound}'"
   end
 end
 
 class Cat < Animal
   def groom
-    puts "lick... lick..."
+    puts 'lick... lick...'
   end
 end
 
 class HouseCat < Cat
   def initialize
-    @sound = "meow"
+    @sound = 'meow'
   end
 end
 
 ##
 # your answers here
-Answer.housecat_noise = nil
-#
+Answer.housecat_noise = "I am a Housecat, and I go 'meow'"
+# HouseCat.new initializes a new Housecat object with the @sound of 'meow'.
+# When we call say_hello, first the Cat class is checked for a say_hello method; it doesn't have one,
+# Animal is checked. Animal has the method, so it is called, with self referring to the object type
+# that called the method, "HouseCat", and @sound referring to the calling objects @sound property.
+
```

Comment:
Pretty sure that was a typo. I feel comfortable with inheritance. 

> On Feb 16, 2016, at 11:14 AM, Matt Brendzel <notifications@github.com> wrote:
> 
> In lib/diagnostic.rb:
> 
> >    end
> >  end
> >  
> >  ##
> >  # your answers here
> > -Answer.housecat_noise = nil
> > -#
> > +Answer.housecat_noise = "I am a Housecat, and I go 'meow'"
> > +# HouseCat.new initializes a new Housecat object with the @sound of 'meow'.
> > +# When we call say_hello, first the Cat class is checked for a say_hello method; it doesn't have one,
> > +# Animal is checked. Animal has the method, so it is called, with self referring to the object type
> > +# that called the method, "HouseCat", and @sound referring to the calling objects @sound property.
> > +
> 
> Otherwise correct. Nice answer.
> 
> —
> Reply to this email directly or view it on GitHub.
>